swift/src/CleanupStaleFlags/StaleFlagCleanerLauncher.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright © Uber Technologies, Inc. All rights reserved.
```

Comment:
Shouldn't this be the OSS license instead? And also have the year/year range?

Reply:
will create a new pull request.